auth0/src/main/AndroidManifest.xml

```diff
@@ -37,7 +37,7 @@
         <activity
             android:name="com.auth0.android.provider.RedirectActivity"
             android:exported="true">
-            <intent-filter>
+            <intent-filter android:autoVerify="true">
```

Comment:
It doesn't break, at least compiler doesn't scream and the apk installs and runs ok. The reason I added is because people using app links would otherwise need to manually override this intent-filter declaration with their own in the manifest.